jest.setup.js

```diff
@@ -0,0 +1,8 @@
+class IntersectionObserver {
```

Comment:
We have `failOnConsole` on the Quizzes UI equivalent of this file. I don't think we have to have it but it might be useful?
https://github.com/Constructor-io/constructorio-ui-quizzes/blob/main/jest.setup.ts

Reply:
Thank you I didn't know I needed this but it will make testing easier for the other PR. Adding it